cfgov/v1/__init__.py

```diff
@@ -35,12 +39,30 @@ def environment(**options):
         'choices_for_page_type': ref.choices_for_page_type,
         'is_blog': ref.is_blog,
         'get_page_state_url': share.get_page_state_url,
+        'external_links': external_links,
     })
     env.filters.update({
         'slugify': slugify,
     })
     return env
 
+EXTERNAL_LINK_PATTERN = '(https?:\/\/(?:www\.)?(?![^\?]*(cfpb|consumerfinance).gov)(?!(content\.)?localhost).*)'
+
+
+def external_links(value):
+    soup = BeautifulSoup(expand_db_html(value.source))
+
+    try:
+        for a in soup('a'):
+            p = re.compile(EXTERNAL_LINK_PATTERN)
```

Comment:
@kave yeah I thought about that but CFPB is controlling the content, so it didn't seem like that'd be an issue. anyway, yes, that sounds good!  :+1: